test-apps/display-test-app/README.md

```diff
@@ -216,7 +216,9 @@ You can use these environment variables to alter the default behavior of various
 * IMJS_NO_ELECTRON_AUTH
   * If defined, the authorization client will not be initialized for the electron app, to work around a current bug that causes it to produce constant exceptions when attempting to obtain an access token.
 * IMJS_USE_FRONTEND_TILES
-  * If defined, the @itwin/frontend-tiles package will be used to obtain tile trees for spatial views, served over localhost.
+  * If defined, specifies url for @itwin/frontend-tiles to obtain tile trees for spatial views, served over localhost.  The string can include special
```

Comment:
Please change name of env var to reflect that it is a URL template, not a boolean.

Reply:
Done